mend_sca_cleanup_tool/sca_cleanup_tool.py

```diff
@@ -54,7 +56,19 @@ def main():
         
     setup_config()
 
-    MAIN_API_CONNECTION = http.client.HTTPSConnection(CONFIG.mend_url)
+    if CONFIG.proxy_host and CONFIG.proxy_port:
+        MAIN_API_CONNECTION = http.client.HTTPConnection(CONFIG.proxy_host, CONFIG.proxy_port)
+        #MAIN_API_CONNECTION.putrequest('CONNECT', CONFIG.mend_url)
+        #MAIN_API_CONNECTION.putheader('Host', CONFIG.mend_url)
+        #MAIN_API_CONNECTION.endheaders()
+        MAIN_API_CONNECTION.set_tunnel(CONFIG.mend_url)
+        #response = MAIN_API_CONNECTION.getresponse()
+        if MAIN_API_CONNECTION._tunnel_host:
```

Comment:
Remove the commented code

Reply:
Let's keep it. I suppose in future we should use it in case of proxy with user name and password. I want to have it here as reminder